dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/h2/V7.0_2019_06_14__scripts-and-process.sql

```diff
@@ -0,0 +1,40 @@
+--
+-- The contents of this file are subject to the license and copyright
+-- detailed in the LICENSE and NOTICE files at the root of the source
+-- tree and available online at
+--
+-- http://www.dspace.org/license/
+--
+
+-- ===============================================================
+-- WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING
+--
+-- DO NOT MANUALLY RUN THIS DATABASE MIGRATION. IT WILL BE EXECUTED
+-- AUTOMATICALLY (IF NEEDED) BY "FLYWAY" WHEN YOU STARTUP DSPACE.
+-- http://flywaydb.org/
+-- ===============================================================
+CREATE SEQUENCE process_id_seq;
+
+CREATE TABLE process
+(
+    process_id              INTEGER NOT NULL PRIMARY KEY,
+    user_id                 UUID NOT NULL,
+    start_time              TIMESTAMP,
+    finished_time           TIMESTAMP,
+    creation_time           TIMESTAMP NOT NULL,
+    script                  VARCHAR(256) NOT NULL,
+    status                  VARCHAR(32),
+    parameters              VARCHAR (512)
```

Comment:
There is an odd looking space here after VARCHAR.. I'm hoping it doesn't cause issues, but we should remove the extra space. This exists in all 3 of the SQL migration scripts.